aws-databrew-job/aws-databrew-job.json

```diff
@@ -0,0 +1,257 @@
+{
+  "typeName": "AWS::DataBrew::Job",
+  "description": "Resource schema for AWS::DataBrew::Job.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-databrew.git",
+  "properties": {
+    "DatasetName": {
+      "description": "Dataset name",
+      "type": "string",
+      "pattern": "[ -\ud7ff\ue000-\ufffd\ud800\udc00-\udbff\udfff\t]*",
+      "minLength": 1,
+      "maxLength": 255
+    },
+    "EncryptionKeyArn": {
+      "description": "Encryption Key Arn",
+      "type": "string",
+      "minLength": 20,
+      "maxLength": 2048
+    },
+    "EncryptionMode": {
+      "description": "Encryption mode",
+      "enum": [
+        "SSE-KMS",
+        "SSE-S3"
+      ],
+      "type": "string"
+    },
+    "Name": {
+      "description": "Job name",
+      "type": "string",
+      "pattern": "[ -\ud7ff\ue000-\ufffd\ud800\udc00-\udbff\udfff\t]*",
```

Comment:
What are these patterns trying to represent?

We're having issues with them in https://github.com/aws-cloudformation/cfn-python-lint/pull/1867